test-dotnet/action.yml

```diff
@@ -56,6 +54,11 @@ runs:
         aws-secret-access-key: ${{inputs.awsAccessKeySecret}}
         aws-region: ${{inputs.awsRegion}}
 
+    # Output event commit ids as an artifact
+    # This is necessary to allow to load the event commits from triggered workflows from this one
+    - name: Output the event commits to an artifact
```

Comment:
Since the test action usually is the source of the ci/cd flows (could be for deploy, nuget, etc), it will store the event commits in the artifact, so it will be available in the subsequent workflows.